src/pages/work/outreach.astro

```diff
@@ -0,0 +1,259 @@
+        due to a highly coupled Ruby and React architecture. Outreach needed to
+        accelerate the replatforming of their primary SaaS product to use more
+        maintainable, faster, and reliable modern technologies.
+      </p>
+
+      <div class="bg-outreach-purple overflow-hidden px-6 rounded-md lg:px-12">
+        <Img
+          alt="Sales execution report detail"
+          attributes={{
+            img: {
+              class: '-mb-6 mt-6 rounded-t-md lg:-mb-12 lg:mt-12'
+            }
+          }}
+          format="webp"
+          placeholder="none"
+          src="/img/work/outreach/sales-execution-report.png"
+        />
+      </div>
+    </div>
+  </section>
+
+  <section
+    class="flex flex-wrap justify-center mt-24 px-8"
+    data-aos="fade-down"
+  >
+    <div class="max-w-6xl w-full">
+      <h2 class="text-4xl mb-6">Solution</h2>
+      <p>
+        Ship Shape's team of experts immediately improved velocity and started
+        delivering features in the updated application. The new application used
+        a microservices API architecture, a custom GraphQL layer, and
+        micro-frontend based apps tied together with an entry single page
+        application. We worked to establish strong communication channels with
+        the team with worldwide distribution.
+      </p>
+
+      <div
+        class="gap-12 grid grid-cols-1 my-12 lg:my-24 lg:gap-24 lg:grid-cols-2"
+      >
+        <div class="lg:order-2">
+          <h3 class="text-2xl mb-6">Desktop Notifications</h3>
+
+          <p>
+            Our team owned the desktop notification feature to implement an
+            application wide strategy for creating, managing, and adding custom
+            actions to notifications. This also resulted in some newly acquired
+            expertise for our team that we shared in [this blog
+            post](https://shipshape.io/blog/add-desktop-notifications/).
+          </p>
+        </div>
+
+        <figure class="lg:order-1">
+          <div
+            class="bg-outreach-purple overflow-hidden px-6 rounded-md lg:px-8"
+          >
+            <Img
+              alt="An action triggered from desktop notification"
+              attributes={{
+                img: {
+                  class: '-mb-6 mt-6 rounded-t-md lg:-mb-8 lg:mt-8'
+                }
+              }}
+              format="webp"
+              placeholder="none"
+              src="/img/work/outreach/desktop-notifications.png"
+            />
+          </div>
+          <figcaption class="italic text-sm">
+            User desktop notification for an urgent task
+          </figcaption>
+        </figure>
+      </div>
+
+      <div
+        class="gap-12 grid grid-cols-1 my-12 lg:my-24 lg:gap-24 lg:grid-cols-2"
+      >
+        <div>
+          <h3 class="text-2xl mb-6">Dynamic customizable table views</h3>
+
+          <p>
+            We added significant functionality to the way Outreach presented
```

Comment:
@chuckcarpenter can we talk about what this extra table functionality was and how it helped users in more detail?